include/db.h

```diff
@@ -1900,24 +1900,30 @@ int controls_link(dbref who, dbref what);
  * With a given owner, name, and source (location).  Returns the DBREF
  * of the exit.  The 'name' memory is copied.
  *
+ * Uses an error parameter to communicate the reason for failure.
```

Comment:
I would rephrase this slightly:

Uses an error parameter to communicate the reason for failure.  This should be at least SMALL_BUFFER_LEN in size.  Obviously this applies to all locations with a similar comment.

Reply:
Good idea! Commit has been pushed.